src/main/java/com/autotune/analyzer/recommendations/updater/vpa/VpaUpdaterImpl.java

```diff
@@ -0,0 +1,371 @@
+                                            .getNamespace())
+                                    .withName(vpaObject
+                                            .getMetadata()
+                                            .getName())
+                                    .patchStatus(vpaObject);
+
+                            LOGGER.info(String.format(AnalyzerConstants.RecommendationUpdaterConstants.InfoMsgs.VPA_PATCHED,
+                                    vpaObject.getMetadata().getName()));
+                        }
+                    }
+                }
+
+            } else {
+                LOGGER.error(AnalyzerErrorConstants.RecommendationUpdaterErrors.UPDATER_NOT_INSTALLED);
+            }
+        } catch (Exception e) {
+            throw new ApplyRecommendationsError(e.getMessage());
+        }
+    }
+
+    /**
+     * This function converts container recommendations for VPA Container Recommendations Object Format
+     */
+    private List<RecommendedContainerResources>  convertRecommendationsToContainerPolicy(HashMap<String, ContainerData> containerDataMap) {
+        List<RecommendedContainerResources> containerRecommendations = new ArrayList<>();
+
+        for (Map.Entry<String, ContainerData> containerDataEntry : containerDataMap.entrySet()) {
+            // fetcing container data
+            ContainerData containerData = containerDataEntry.getValue();
+            String containerName = containerData.getContainer_name();
+            HashMap<Timestamp, MappedRecommendationForTimestamp> recommendationData = containerData.getContainerRecommendations().getData();
+
+            // checking if recommendation data is present
+            if (recommendationData != null) {
+                for (MappedRecommendationForTimestamp value : recommendationData.values()) {
+                    /*
+                     * Fetching Short Term Cost Recommendations By Default
+                     * TODO:// Implement functionality to choose the desired term and model
+                     **/
+                    TermRecommendations termRecommendations = value.getShortTermRecommendations();
+                    HashMap<AnalyzerConstants.ResourceSetting,
+                            HashMap<AnalyzerConstants.RecommendationItem,
+                                    RecommendationConfigItem>> recommendationsConfig = termRecommendations.getCostRecommendations().getConfig();
+
+                    Double cpuRecommendationValue = recommendationsConfig.get(AnalyzerConstants.ResourceSetting.requests).get(AnalyzerConstants.RecommendationItem.CPU).getAmount();
+                    Double memoryRecommendationValue = recommendationsConfig.get(AnalyzerConstants.ResourceSetting.requests).get(AnalyzerConstants.RecommendationItem.MEMORY).getAmount();
+
+                    LOGGER.info(String.format(AnalyzerConstants.RecommendationUpdaterConstants.InfoMsgs.RECOMMENDATION_VALUE, "CPU", containerName, cpuRecommendationValue));
```

Comment:
Can strings like `CPU` and `MEMORY` be replaced by constants?

Reply:
Sure, Missed that earlier that we already have constants defined for `CPU` and `MEMORY`. Updated the code to use `constants`.

Reply:
Thanks for the changes!